eosdis-nasa/earthdata-pub-api
# Edpub 1269 enable ornl metadata step

# Description

Adds support for ORNL push metadata step

## Spec

See Ticket: [EDPUB-1269](https://bugs.earthdata.nasa.gov/browse/EDPUB-1269)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Deploy to SIT
3. Confirm ORNL workflow can be run to completion.

---

## Change Log

Adds support for ORNL push metadata step

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
